app/assets/stylesheets/application.css

```diff
@@ -17,7 +17,7 @@
 input,
 textarea,
 .ts-control {
-  box-shadow: inset -1px -1px 2px 0 rgba(13, 148, 137, 0.94) !important;
+  box-shadow: 1px 2px 4px 1px rgba(206, 205, 205, 0.94) !important;
```

Comment:
I can't really tell what the difference is here 🤷

Reply:
The shadow instead of going inwards into the input now it's just outward. I changed the colors from tea to simple gray and just changed the size of the shadow and blur.